mailpoet/assets/js/src/automation/integrations/mailpoet/steps/send-email/edit/email-panel.tsx

```diff
@@ -95,6 +97,12 @@ export function EmailPanel(): JSX.Element {
           )
         }
       />
+      {window.mailpoet_mss_active &&
```

Comment:
Not a blocker, but why is this information in `window` and not in the context?

Reply:
It is always loaded in the window object in every page. Coming from [PageRenderer](https://github.com/mailpoet/mailpoet/blob/5cd7dad6268f0735082ee856b368dfe5e4afe615/mailpoet/lib/AdminPages/PageRenderer.php#L183).